src/Symfony/Component/Intl/Tests/LanguagesTest.php

```diff
@@ -978,11 +1572,12 @@ public function testGetAlpha3Names($displayLocale)
 
         $alpha3Codes = array_keys($names);
         sort($alpha3Codes);
-        $this->assertSame(array_values(self::$alpha2ToAlpha3), $alpha3Codes);
+        $this->assertSame(self::$alpha3Codes, $alpha3Codes);
 
         $alpha2Names = Languages::getNames($displayLocale);
         foreach ($alpha2Names as $alpha2Code => $name) {
-            if (!isset(self::$alpha2ToAlpha3[$alpha2Code])) {
+            $alpha3Code = self::$alpha2ToAlpha3[$alpha2Code] ?? (3 === strlen($alpha2Code) ? $alpha2Code : null);
```

Comment:
Please remove these lines from the test (Line 1578- 1583).

The list of names returned from `Languages::getAlpha3Names` should be the same as the list of names returned from `Languages::getNames`, just with possible different array keys.

Reply:
improved test similar like `getNames`